config.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+
+class Config(object):
+    SECRET_KEY = os.environ.get('SECRET_KEY')
+    SQLALCHEMY_DATABASE_URI = os.environ.get('DB_URI')
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
```

Comment:
Since this is a Python file, by flake8 standards you should add a new line at this one too.